### PR TITLE
chore: upgrade codewhisperer-streaming client

### PR DESCRIPTION
**Pull request by JuliaABurch**
## Problem
The streaming client needs to be updated to access recent API changes.

## Solution
I followed the instructions [here](https://github.com/aws/aws-toolkit-vscode/blob/master/docs/build.md) to regenerate the client source files. I ran a quick test of basic chat features (Q, Featuredev, Gumby) to ensure stability.

~~Note: the client is 2 months out of date at this point, so quite a few changes have been generated.~~

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by tincheng — APPROVED** [review]

**Comment by justinmk3** [comment]
1 million added lines and almost no deletions seems wrong. Please look at previous PRs to see which files actually need to be added.

**Comment by JuliaABurch** [comment]
> 1 million added lines and almost no deletions seems wrong.
> 
> Probably the first step, is to delete the existing directory and replace it with the new generated result. That should result in a much more reasonable diff.
> 
> If that's not mentioned in the instructions, please update the instructions.

I've updated the instructions with some error handling so that the clean / deletion happens much more reliably.

File: src.gen/@amzn/codewhisperer-streaming/package-lock.json
Comment (justinmk3) [inline src.gen/@amzn/codewhisperer-streaming/package-lock.json:1]: I don't think `src.gen/@amzn/codewhisperer-streaming/package-lock.json` should be committed? None of our other sub projects have package-lock.json , and it wasn't needed before. 

The locked packages for this sub-project are already reflected in the top-level package-lock.json. It's possible that some changes are missing from that file, in this PR.

Comment (justinmk3) [reply src.gen/@amzn/codewhisperer-streaming/package-lock.json:1]: I don't see any dependency changes in `src.gen/@amzn/codewhisperer-streaming/package.json`, so the top-level package-lock.json probably doesn't have any changes.

So all that's needed is to remove `src.gen/@amzn/codewhisperer-streaming/package-lock.json` from this PR.

**Pull request merged**